pdf_parser.py

```diff
@@ -0,0 +1,40 @@
+import fitz
+from common_utils import *
+
+# PDF files to png pages parser
+class PDF_parser:
+    def __init__(self, output_folder=os.environ['FOLDER_PAGES']):
+        self.page_output_folder = output_folder
+
+        self.pdf_file_list = []
+
+        if not os.path.exists(self.page_output_folder):
+            os.makedirs(self.page_output_folder)
+
+    # load PDF, go through pages and save them as PNG
+    def pdf_to_png(self, pdf_file: str, large_size=True) -> None:
+        items = [page_file for page_file in os.listdir(self.page_output_folder) if
+                 page_file.startswith(pdf_file.split(".")[-2])]
+        if len(items) == fitz.open(pdf_file).page_count:
+            print(f"{pdf_file} was already split into {len(items)} page image(s)")
+            return
+
+        # high resolution image save
+        zoom = 2.0 if large_size else 1.0
+        mat = fitz.Matrix(zoom, zoom)
+
+        doc = fitz.open(pdf_file)
+        file_name = pdf_file.split("/")[-1].split(".")[0]
```

Comment:
Does not work on Windows. Use `pathlib.Path`.

Reply:
fixed